assets/kube-burner-report-ocp-wrapper/queries.libsonnet

```diff
@@ -499,9 +499,9 @@ local elasticsearch = g.query.elasticsearch;
         + elasticsearch.withQuery(query)
         + elasticsearch.withTimeField('timestamp'),
     queries(): [
-        self.base("Namespaces", "uuid.keyword: $uuid AND metricName: \"namespaceCount\" AND labels.phase: \"Active\""),
-        self.base("Services", "uuid.keyword: $uuid AND metricName: \"serviceCount\""),
-        self.base("Deployments", "uuid.keyword: $uuid AND metricName: \"deploymentCount\""),
+        self.base("Namespaces", "uuid.keyword: $uuid AND metricName.keyword: \"namespaceCount\" AND labels.phase: \"Active\""),
```

Comment:
If we're modifying this dashboard, I also find that using "max" instead of "average" for the aggregatesCount panel is better when we collect garbage collection metrics. (Looks like it might be line 495 above, but I can't put a comment there.)

Otherwise, the count shown is an average of the scale up & down.

Reply:
Acknowledged. Will take that in a different PR. Please feel free to file an issue or RFE.